yogo/dawlets/algolet/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "algolet",
+  "name": "@dawlet/algolet",
```

Comment:
I reserved the `@dawlet` organizatino on npm.

@zigen @szk2s 
I will send you guys invitations when needed